pkg/api/convertertoapi_test.go

```diff
@@ -918,9 +920,9 @@ func TestConvertVLabsWindowsProfile(t *testing.T) {
 	}
 
 	for _, c := range cases {
+		c := c
```

Comment:
I would suggest to give it a new variable name.
say localCase.

Reply:
localCase := c

and then use localCase inner the t.Runblabla

Reply:
Good suggestion. We could change it later in master. Since this is a cherry-pick, let us keep it in this PR.

Reply:
And you also can see the example in the reference: https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks. The name should be ok.